### PR TITLE
Sh/action.yml

### PR DESCRIPTION
**Pull request by sshjelle**
# Changes
- action.yml is tested and works
- add github/md2confluence.yml - tested and works
- add github/md2confluenceUsingAction.yml - tested and works

### CODE REVIEW

File: action.yml
Comment (ottolote) [inline action.yml]: could change this to false, default is working dir 👍 
```suggestion
    required: false
```

**Review by ottolote — DISMISSED** [review]
Looks good! 👍 

Putting the `github` folder in `docs/` might make sense but I'll leave it up to you

**Comment by sshjelle** [comment]
@ottolote, pushed the changes you suggested, need another approval before merge

**Review by ottolote — APPROVED** [review]
👍

**Pull request merged**
